src/main/java/vazkii/botania/client/integration/jei/orechid/OrechidIgnemRecipeCategory.java

```diff
@@ -0,0 +1,95 @@
+/**
+ * This class was created by <codewarrior0>. It's distributed as
+ * part of the Botania Mod. Get the Source Code in github:
+ * https://github.com/Vazkii/Botania
+ * <p/>
+ * Botania is Open Source and distributed under the
+ * Botania License: http://botaniamod.net/license.php
+ */
+package vazkii.botania.client.integration.jei.orechid;
+
+import mezz.jei.api.IGuiHelper;
+import mezz.jei.api.gui.IDrawable;
+import mezz.jei.api.gui.IDrawableStatic;
+import mezz.jei.api.gui.IGuiItemStackGroup;
+import mezz.jei.api.gui.IRecipeLayout;
+import mezz.jei.api.ingredients.IIngredients;
+import mezz.jei.api.recipe.IRecipeCategory;
+import mezz.jei.api.recipe.IRecipeWrapper;
+import net.minecraft.client.Minecraft;
+import net.minecraft.client.renderer.GlStateManager;
+import net.minecraft.client.resources.I18n;
+import net.minecraft.item.ItemStack;
+import net.minecraft.util.ResourceLocation;
+import vazkii.botania.common.item.block.ItemBlockSpecialFlower;
+import vazkii.botania.common.lib.LibBlockNames;
+import vazkii.botania.common.lib.LibMisc;
+
+import javax.annotation.Nonnull;
+
+public class OrechidIgnemRecipeCategory implements IRecipeCategory {
+
+    public static final String UID = "botania.orechid_ignem";
+    private final IDrawableStatic background;
+    private final String localizedName;
+    private final IDrawableStatic overlay;
+
+    public OrechidIgnemRecipeCategory(IGuiHelper guiHelper) {
+        background = guiHelper.createBlankDrawable(168, 64);
+        localizedName = I18n.format("botania.nei.orechidIgnem");
+        overlay = guiHelper.createDrawable(new ResourceLocation("botania", "textures/gui/pureDaisyOverlay.png"),
+                0, 0, 64, 46);
+    }
+
+    @Nonnull
+    @Override
+    public String getUid() {
+        return UID;
+    }
+
+    @Nonnull
+    @Override
+    public String getTitle() {
+        return localizedName;
+    }
+
+    @Nonnull
+    @Override
+    public IDrawable getBackground() {
+        return background;
+    }
+
+
+    @Override
+    public void setRecipe(IRecipeLayout recipeLayout, IRecipeWrapper recipeWrapper, IIngredients ingredients) {
+        final IGuiItemStackGroup itemStacks = recipeLayout.getItemStacks();
+
+        itemStacks.init(0, true, 40, 12);
+
+        itemStacks.set(0, ingredients.getInputs(ItemStack.class).get(0));
+
+        itemStacks.init(1, true, 70, 12);
+        itemStacks.set(1, ItemBlockSpecialFlower.ofType(LibBlockNames.SUBTILE_ORECHID_IGNEM));
+
+        itemStacks.init(2, true, 99, 12);
+
+        itemStacks.set(2, ingredients.getOutputs(ItemStack.class).get(0));
+

```

Comment:
make your whitespace consistent between init and set (here and the other categories too)

Reply:
Got it.